docs/CHANGELOG.md

```diff
@@ -24,10 +24,12 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 * `Material` animations now directly use `Handle<Texture>` instead of using indirection. ([#1089])
 * `SpriteRenderPrimitive::SpriteSheet` now takes `Handle<SpriteSheet>` instead of a `u64` ID. ([#1089])
+* `nalgebra` is now the math library used by the engine ([#1066])
 
 ### Removed
 
 * `SpriteSheetSet` is removed as it is no longer needed. ([#1089])
+* `amethyst::core::orientation` has been removed because of limited use compared to the `Transform` component.
```

Comment:
s/orientation/Orientation

Also, it was meant to be used **with** Transform, and not separately.

Reply:
This is a user facing changelog, no? If isn't then sure, I can list the source path instead but otherwise I would like to keep the import path.